fdbserver/workloads/BackupCorrectness.actor.cpp

```diff
@@ -757,7 +755,7 @@ struct BackupAndRestoreCorrectnessWorkload : TestWorkload {
 					printf("BackupCorrectness, backupAgent.restore is called for restoreIndex:%d tag:%s\n",
 					       restoreIndex,
 					       restoreTag.toString().c_str());
-					restores.push_back(backupAgent.restore(cx,
+					restores.push_back(backupAgent.restore(self->extraDB,
```

Comment:
When we restore to the extraDB we should make sure consistency checks (Cycle, AtomicOps) are run on this new DB